jax/_src/pallas/mosaic/lowering.py

```diff
@@ -1793,6 +1793,34 @@ def _neg_lowering_rule(ctx: LoweringRuleContext, x):
 skip_mlir_conversions.add(lax.neg_p)
 
 
+def _sign_lowering_uint_helper(x):
```

Comment:
Consider inlining all the helpers and calling `mlir.lower_fun` on the whole lowering rule instead of on each helper separately.

Reply:
Done